### ISSUE TITLE
Service fabric definition not working

### ISSUE THREAD

**Issue by miker1423** [Microsoft/vsts-tasks#3624]
Hello, I'm trying to build a Service Fabric project that contains some services built on .Net Core targeting .Net 452, when the build definition triggers to build it fails with the following error message:

`
Error MSB4041: The default XML namespace of the project must be the MSBuild XML namespace. If the project is authored in the MSBuild 2003 format, please add xmlns="http://schemas.microsoft.com/developer/msbuild/2003" to the <Project> element. If the project has been authored in the old 1.0 or 1.2 format, please convert it to MSBuild 2003 format.
` 

Is there a work around to make this work?

**Comment by bishalpd** [comment]
@miker1423 can you please provide more details:

- are you using hosted agent?

- which VS version have you selected in the build sfproj task - VS2015 or VS2017?

- Please provide more detailed error log by adding a variable "system.debug" with value true in build definition and creating a new build

- Would be great if you can provide repro steps/sample app to repro this issue

**Comment by miker1423** [comment]
Hello, 
- I'm using hosted agent
- VS2017
- Even adding the variable "system.debug" I get the same  error
- I'm just picking the template that is found in VS Team Services, the only thing I change is the version of VS from 2015 to 2017, and the architecture from x86 to x64.

**Comment by bishal-pdMSFT** [comment]
@miker1423 i think this is same issue as https://github.com/Microsoft/vsts-tasks/issues/3747
@bryanmacfarlane, do you agree?
Hosted agent does not support VS2017. A new queue called "Hosted agent VS2017" is being rolled out which will fix this.

**Comment by miker1423** [comment]
@bishal-pdMSFT yes, it's the same issue, do you have more info on when its going to roll out the new queue??

**Comment by bishal-pdMSFT** [comment]
@miker1423 It is already being rolled out. Please re-check for your account.

**Comment by miker1423** [comment]
Hello, I'm now getting this other error when building the .sfproj

```xml
Error MSB4057: The target "Package" does not exist in the project.
```

**Comment by bishal-pdMSFT** [comment]
@miker1423 @bishal-pdMSFT This could be happening as your sfproj might be older than minimum supported version by SF sdk and tools running on HostedVS2017 agent. Current version on this queue is 5.7-preview.
You should install this version of SF sdk and runtime on your local machine and update sfproj. I think you might not be able to install 5.7-preview version as it is not available on WebPI, but you an try 5.6 version and see if that fixes your issue

**Comment by bishal-pdMSFT** [comment]
Closing this. @miker1423, please re-activate if you still have this issue.
